source/includes/_connected-apps.md

```diff
@@ -260,3 +260,118 @@ It is possible that a user's refresh token will become invalid. This could happe
+
+1. At some point before going live to your customers, you set up an "application" webhook subscription for verification status change events. This is a single subscription that will allow you to receive verification status events related to any profile that links to your application in the future. (See [here](#application-webhooks) for more information.)
+1. After going live, a user connects their profile to your application using the OAuth authorization flow. (Note that the profile might be an existing, verified profile, or a brand new, unverified profile).
+2. During the authorization flow, your system receives an authorization code and TransferWise profile ID in a request to your callback endpoint (see [User authorization](#connected-apps-integration-guide-user-authorization)). Once API tokens have been acquired using the authorization code, you now have API tokens and the related profile ID: you can now access the profile's API resources.
+4. The verification status of the profile is queried using the TransferWise API. If the response indicates the profile is "verified", you can proceed with transfer operations. If the response indicates "not verified", then transfers are likely to be delayed: you should wait to receive a webhook notification for the customer.
+5. If a profile is unverified, the verification process takes place. Later, when verification completes successfully, you will receive a webhook notification informing you that the profile has become verified: you may then allow transfer creation. (Alternatively, you can continue to poll the API again, but this is not recommended.)
+
+
+### Checking verification status using the API
+
+You can check the verification status of a profile using the following API.
+This is a profile-specific API resource which should be accessed using an access token acquired for the profile.
+
+> Example Request:
+
+```shell
+    curl -X GET 'https://api.sandbox.transferwise.tech/v3/profiles/12345/verification-status' \
+        -H "Authorization: Bearer <profile's access token>"
+```
+
+### Request
+
+**`GET https://api.sandbox.transferwise.tech/v3/profiles/{profileId}/verification-status`**
+
+Field           | Description       | Format
+-----           | -----------       | ------
+profileId       | Profile ID        | Integer
+
+> Example Response:
+
+```json
+{
+  "profileId": 12345,
+  "currentStatus": "verified"
+}
+```
+
+### Response
+
+Field           | Description                   | Format
+-----           | -----------                   | ------
+profileId       | Profile ID                    | Integer
+currentStatus   | Current verification status   | Verification status code (see below)
+
+Verification status code    | Description
+------------------------    | -----------
+`verified`                  | Profile is sufficiently verified
+`not_verified`              | Profile is not sufficiently verified
```

Comment:
How about these descriptions, or something like this?
```
verified = The profile is sufficiently verified to start making payments (note: some regional limits may still apply)
not_verified = The profile is currently awaiting verification or there are pending issues with the verification process
```

Reply:
Yep, I will make these changes.